### PR TITLE
Update Python Version for local gen_module

### PR DESCRIPTION
**Pull request by UuuNyaa**
### Purpose of the pull request

Due to the update of the Python version from 3.8 to 3.9 in GitHub Actions dependencies in commit 782a482d4eb5e48ab932cc0e3d60fcbbcb435a72, we need to update the Python version locally to align with the GitHub Actions environment.

### Description about the pull request

- Update the Python version in `tools/gen_module/Dockerfile` from 3.8 to 3.9.
- Correct the incorrect script file name in `docs/generate_modules.md`.

### CODE REVIEW

**Comment by nutti** [comment]
@JonathanPlasse 

This PR fixes the Python version.
Should we wait your WIP PR #207?
I think your PR changes the supported Python version.

**Comment by JonathanPlasse** [comment]
Pyi files should be compatible for all Python version, as they are never executed.
The PR already use Python 3.10 features in the Pyi files, but the CI passes as expected.

**Comment by nutti** [comment]
@JonathanPlasse 

Hmm...
I'm not sure why CI passes even if you use Python 3.10 feature.
But, I think we should update the requirement to use Python 3.10 from your PR.
Is it possible to update Python version to 3.10 in your PR?

**Comment by nutti** [comment]
@UuuNyaa 

Thank you for your contribution again!
Could you update your code to use Python 3.10 from above discussion?

**Comment by nutti** [comment]
@JonathanPlasse 

Ah, I got it.
Thank you for your explanation.

File: tools/gen_module/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.8-bullseye
+FROM python:3.9-bullseye
```
Comment (nutti) [inline tools/gen_module/Dockerfile:1]:
```suggestion
FROM python:3.10-bullseye
```

**Review by nutti — CHANGES_REQUESTED** [review]

**Comment by UuuNyaa** [comment]
@nutti @JonathanPlasse Thanks for the comments and review!

I have confirmed that this change works correctly in my environment.
```diff
- FROM python:3.9-bullseye
+ FROM python:3.10-bullseye
```

**Pull request merged**